sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/Coverage.java

```diff
@@ -34,7 +33,6 @@
   private static final int GROW_FACTOR = 2;
   private static final int SPECIAL_HITS_NON_EXECUTABLE = -1;
   private final Map<String, int[]> hitsByLineAndFile = new HashMap<>();
-  private final Map<String, List<BranchCoverage>> branchCoverageByFile = new HashMap<>();
   private final List<BranchPoint> branchPoints = new ArrayList<>();
```

Comment:
is this needed?

Reply:
I'm not sure I understand what you mean. In this case the variable was not used anymore since we no longer push branch coverage this way so it can be removed.

Reply:
exactly - so I think you can remove what's not used anymore, unless you reckon it's going to be used in the next PR

Reply:
my comment is on this line
```
   private final List<BranchPoint> branchPoints = new ArrayList<>(); 
```

Reply:
We discussed directly, but I will add a comment also here to not forget about it. This is needed in order to properly compute the imported branch coverage.